aggregations/db_tables/deployed_contracts.py

```diff
@@ -32,21 +45,29 @@ def sql_select(self):
         return """
             SELECT
                 action_receipt_actions.args->>'code_sha256' as contract_code_sha256,
-                receipts.receiver_account_id as deployed_to_account_id,
-                receipts.receipt_id as deployed_by_receipt_id,
-                receipts.included_in_block_timestamp as deployed_at_block_timestamp
+                action_receipt_actions.receipt_receiver_account_id as deployed_to_account_id,
+                action_receipt_actions.receipt_id as deployed_by_receipt_id,
+                execution_outcomes.executed_in_block_timestamp as deployed_at_block_timestamp,
+                execution_outcomes.executed_in_block_hash as deployed_at_block_hash
             FROM action_receipt_actions
-            JOIN receipts ON receipts.receipt_id = action_receipt_actions.receipt_id
-            WHERE action_kind = 'DEPLOY_CONTRACT'
-                AND receipts.included_in_block_timestamp >= %(from_timestamp)s
-                AND receipts.included_in_block_timestamp < %(to_timestamp)s
-            ORDER BY receipts.included_in_block_timestamp
+            JOIN execution_outcomes ON execution_outcomes.receipt_id = action_receipt_actions.receipt_id
+            WHERE action_receipt_actions.action_kind = 'DEPLOY_CONTRACT'
+                AND execution_outcomes.status = 'SUCCESS_VALUE'
+                AND execution_outcomes.executed_in_block_timestamp >= %(from_timestamp)s
+                AND execution_outcomes.executed_in_block_timestamp < %(to_timestamp)s
+            ORDER BY execution_outcomes.executed_in_block_timestamp
         """
 
     @property
     def sql_insert(self):
         return """
-            INSERT INTO deployed_contracts VALUES %s 
+            INSERT INTO deployed_contracts (
+                contract_code_sha256,
+                deployed_to_account_id,
+                deployed_by_receipt_id,
+                deployed_at_block_timestamp,
+                deployed_at_block_hash
+            ) VALUES %s
```

Comment:
You don't need to mention all the fields here, you can just leave the previous version, it should work fine

Reply:
I prefer to be explicit, so it is easier to see what are the values and in which order they should be.